hircluster.c

```diff
@@ -3329,6 +3363,89 @@ int redisClusterAppendCommand(redisClusterContext *cc, const char *format,
     return ret;
 }
 
+/* Get the first slot served by a given node */
+static int __redisClusterGetServedSlot(cluster_node *node) {
```

Comment:
Is this still needed?

Reply:
Did some changes of this and will push an additional commit to avoid loosing history with a force-commit.
Legacy the slot is used to get the hiredis context, but for the ToNode I now added a variant to get it via the address.